bin/handler-slack-multichannel.rb

```diff
@@ -301,7 +321,7 @@ def payload(notice, channel)
     {
       icon_url: slack_icon_url ? slack_icon_url : 'https://raw.githubusercontent.com/sensu/sensu-logo/master/sensu1_flat%20white%20bg_png.png',
       attachments: [{
-        title: "#{@event['client']['name']} - #{translate_status}",
+        title: "#{@event['check']['name']} - #{translate_status}",
```

Comment:
this makes it technically a breaking change even though from a glance it looks reasonable. I will need to review this further when its not so late.

Reply:
Seems like the original author of this multi-channel version (who was working in our company) changed from `client` to `check` at some point since it made a lot more sense to our monitoring via slack.
In our current setup (using `check` here) the client name is appended at the end of the message so having it in the title is redundant.
Personally, I think it would make sense to have both the client and the check in the title of the message, since that's what I want to be aware of when I read the notifications on slack.
I am however willing to change this back if you think it is a breaking change.

Reply:
I would not be opposed to having something like `client/check` only reason I said its breaking is because it removes the original message and could lead to confusion. I think this enhances the message in a positive way that reduces confusion. We might want to consider replacing the client name with the source if present.

```suggestion
        title: "#{@event['client']['name']}/#{@event['check']['name']} - #{translate_status}",
```